udata/models/__init__.py

```diff
@@ -180,8 +180,10 @@ def init_app(app):
     for plugin in app.config['PLUGINS']:
         name = 'udata.ext.{0}.models'.format(plugin)
         try:
-            __import__(name)
-        except ImportError:
-            pass
-        except Exception as e:
+            module_obj = importlib.import_module(name)
+            for item in getattr(module_obj, '__all__'):
+                globals()[item] = getattr(module_obj, item)
+        except ImportError as e:
```

Comment:
Maybe in debug only because not all modules exports models and in this case the ImportError is expected and we dont want to spam the logs.